src/Pim/Bundle/CatalogBundle/Elasticsearch/ProductIndexer.php

```diff
@@ -64,6 +66,33 @@ public function indexAll(array $products, array $options = [])
         }
 
         $this->indexer->bulkIndexes($this->indexType, $normalizedProducts, 'id');
+
+        if (!isset($options['associated_products']) || false === $options['associated_products']) {
+            $this->indexAssociatedProducts($products);
+        }
+    }
+
+    /**
+     * @param ProductInterface[] $products
+     */
+    private function indexAssociatedProducts(array $products)
```

Comment:
indexProductsAssociatedToTheFollowingProducts
?
or something like that
or should be explained in the comments